WooCommerce/Classes/ViewRelated/Orders/OrdersRootViewController.swift

```diff
@@ -153,12 +155,18 @@ final class OrdersRootViewController: UIViewController {
 
     /// Presents the Order Creation flow.
     ///
-    @objc func presentOrderCreationFlow() {
+    /// - parameter initialItem: Represents the product ID of an initial item to create the Order with, if given
```

Comment:
Refactoring missed this bit:
```suggestion
    /// - parameter initialProductID: Represents the product ID of an initial item to create the Order with, if given
```

Reply:
Done on https://github.com/woocommerce/woocommerce-ios/pull/9736/commits/244b26d3b1881440084c0c197d69b8edb2201588